File: java/Aksesoris.java
public class Aksesoris extends PetShop {
    // Additional attributes
    private String jenis;
    private String bahan;
    private String warna;
    
    // Constructor
    public Aksesoris(int id, String nama_produk, int harga_produk, int stok_produk, 
                     String jenis, String bahan, String warna) {
        // Call the parent constructor
        super(id, nama_produk, harga_produk, stok_produk);
        
        // Initialize additional attributes
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }
    
    // Getter methods
    public String getJenis() {
        return jenis;
    }
    
    public String getBahan() {
        return bahan;
    }
    
    public String getWarna() {
        return warna;
    }
    
    // Setter methods
    public void setJenis(String jenis) {
        this.jenis = jenis;
    }
    
    public void setBahan(String bahan) {
        this.bahan = bahan;
    }
    
    public void setWarna(String warna) {
        this.warna = warna;
    }
}

File: python/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    # Constructor
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna):
        # Memanggil constructor dari parent class (PetShop)
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        
        # Atribut tambahan untuk Aksesoris
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna
    
    # Getter methods
    def get_jenis(self):
        return self.__jenis
    
    def get_bahan(self):
        return self.__bahan
    
    def get_warna(self):
        return self.__warna
    
    # Setter methods
    def set_jenis(self, jenis):
        self.__jenis = jenis
    
    def set_bahan(self, bahan):
        self.__bahan = bahan
    
    def set_warna(self, warna):
        self.__warna = warna